### PR TITLE
(WIP) Fix #1271: remove inclusion of stack trace for logging

### LINKED ISSUE #1271: Remove stack trace from logging of "known exceptions" in `RequestHandler`, `AuthnJwtService`
There are a few cases in which some of APIs both log an exception and throw it: this is usually frowned upon, but may make sense for gathering some information. In such cases it is, however, usually best not to always include stack trace in logging -- especially when exception type and message indicate actual reason.
This is the case for a few Authnz handlers, as well as code that resolves C* resources such as keyspaces and tables.

I'll do a PR that will remove places where stack trace looks unnecessary, based on existing CI log inspection. This should further improve readability of the log output by removing more fluff.

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Main thing is to remove inclusion of stack trace for selected cases of logging, where trace adds no valuable information (where we know from context, exception type and message all pertitent info).

Also removes one extra Maven dependency that mvn build complained about.

**Which issue(s) this PR fixes**:
Fixes #1271

**Checklist**
- [x] Changes manually tested -- check out CI logs to verify (compared to earlier)
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated

### CODE REVIEW

File: restapi/src/main/java/io/stargate/web/resources/RequestHandler.java [hunk 1]
```diff
@@ -38,29 +38,31 @@ public static Response handle(Callable<Response> action) {
...
-      logger.info("Bad request", iae);
+      logger.info("Bad request (IllegalArgumentException->BAD_REQUEST): {}", iae.getMessage());
       return Response.status(Response.Status.BAD_REQUEST)
           .entity(
               new Error(
                   "Bad request: " + iae.getMessage(), Response.Status.BAD_REQUEST.getStatusCode()))
           .build();
     } catch (InvalidRequestException ire) {
-      logger.info("Bad request", ire);
+      logger.info("Bad request (InvalidRequestException->BAD_REQUEST): {}", ire.getMessage());
```
Comment (dougwettlaufer) [inline restapi/src/main/java/io/stargate/web/resources/RequestHandler.java:56]: Isn’t the `BAD_REQUEST` redundant?

**Review by dougwettlaufer — APPROVED** [review]
Minor nitpick on log message but looks good otherwise

File: restapi/src/main/java/io/stargate/web/resources/RequestHandler.java (hunk 1 above)
Comment (tatu-at-datastax) [reply restapi/src/main/java/io/stargate/web/resources/RequestHandler.java:56]: It is; I am not sure what'd be most meaningful combination here. Intent is/was to indicate Exception gotten, HTTP response given, and do that consistently. So I am definitely open to a better scheme.

File: persistence-test/src/main/java/io/stargate/it/storage/ExternalStorage.java [hunk 2]
```diff
@@ -293,7 +293,7 @@ public void stop() {
           }
         } catch (Exception e) {
           // This should not affect test result validity, hence logging as WARN
-          LOG.warn("Exception during CCM cluster shutdown: {}", e, e);
+          LOG.warn("Exception during CCM cluster shutdown: " + e.getMessage(), e);
```
Comment (tatu-at-datastax) [inline persistence-test/src/main/java/io/stargate/it/storage/ExternalStorage.java:296]: Would have failed to do what is expected: overload was taking both `e`s as substitutions (but intent was to include stack trace).

File: restapi/src/main/java/io/stargate/web/resources/RequestHandler.java (hunk 1 above)
Comment (tatu-at-datastax) [reply restapi/src/main/java/io/stargate/web/resources/RequestHandler.java:56]: Will merge as-is but can easily improve as a follow-up if we can figure out better scheme here.

**Pull request merged**